Fix build to work with SPARC OpenGL
Need to determine if still necessary

--- a/xdriinfo.c	Tue May 13 12:10:12 2008
+++ b/xdriinfo.c	Tue May 13 12:10:19 2008
@@ -24,7 +24,6 @@
 
 #define GLX_GLXEXT_PROTOTYPES
 #include <GL/glx.h>
-#include <GL/glxext.h>
 #include <X11/Xlib.h>
 #include <stdio.h>
 #include <unistd.h>
